lib/airbrake/rack/user.rb

```diff
@@ -26,8 +26,13 @@ def self.try_current_user(rack_env)
         controller = rack_env['action_controller.instance']
         return unless controller.respond_to?(:current_user, true)
         return unless [-1, 0].include?(controller.method(:current_user).arity)
-        controller.__send__(:current_user)
+        begin
+          controller.__send__(:current_user)
+        rescue Exception => _e # rubocop:disable Lint/RescueException
+          nil
+        end
       end
+
```

Comment:
Sorry, I don't want to sound nitpicky but let's not add this line. I've been using this style consistently across the codebase, so that it's easy to see that the private method is just above.

Reply:
sure, no problem